Specs/DateTime/Dutch/DateTimePeriodParser.json

```diff
@@ -1723,72 +1972,121 @@
+          }
+        },
+        "Start": 92,
+        "Length": 23
+      }
+    ]
   },
   {
-    "Input": "Hi Cortana- maak een skype meeting met Jennifer aan. Ik wil volgende vrijdagmiddag een afspraak van 30 minuten!",
+    "Input": "Cortane, maak een afspraak met skype for business met Wayne op vrijdagmiddag tussen 1 en 4. ",
     "Context": {
-      "ReferenceDateTime": "2017-11-13T16:12:00"
+      "ReferenceDateTime": "2017-11-14T19:12:00"
     },
+    "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "volgende vrijdagmiddag",
+        "Text": "vrijdagmiddag tussen 1 en 4",
         "Type": "datetimerange",
         "Value": {
-          "Timex": "2017-11-24TAF",
+          "Timex": "(XXXX-WXX-5T1,XXXX-WXX-5T4,PT3H)",
           "FutureResolution": {
-            "startDateTime": "2017-11-24 12:00:00",
-            "endDateTime": "2017-11-24 16:00:00"
+            "startDateTime": "2017-11-17 01:00:00",
+            "endDateTime": "2017-11-17 04:00:00"
           },
           "PastResolution": {
-            "startDateTime": "2017-11-24 12:00:00",
-            "endDateTime": "2017-11-24 16:00:00"
+            "startDateTime": "2017-11-10 01:00:00",
+            "endDateTime": "2017-11-10 04:00:00"
           }
         },
-        "Start": 60,
-        "Length": 22
+        "Start": 63,
+        "Length": 27
       }
-    ],
-    "NotSupported": "dotNet",
-    "NotSupportedByDesign": "javascript,python,java"
+    ]
   },
   {
-    "Input": " Hi Cortana- maak een skype meeting met Jennifer aan. Ik wil een afspraak van 30 minuten op afgelopen vrijdagmiddag! ",
+    "Input": "Cortane, maak een afspraak met skype for business met Wayne op vrijdagmiddag tussen 13 en 16. ",
     "Context": {
-      "ReferenceDateTime": "2017-11-13T16:12:00"
+      "ReferenceDateTime": "2017-11-14T19:12:00"
     },
+    "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "afgelopen vrijdagmiddag",
+        "Text": "vrijdagmiddag tussen 13 en 16",
         "Type": "datetimerange",
         "Value": {
-          "Timex": "2017-11-10TAF",
+          "Timex": "(XXXX-WXX-5T13,XXXX-WXX-5T16,PT3H)",
           "FutureResolution": {
-            "startDateTime": "2017-11-10 12:00:00",
-            "endDateTime": "2017-11-10 16:00:00"
+            "startDateTime": "2017-11-17 13:00:00",
+            "endDateTime": "2017-11-17 16:00:00"
           },
           "PastResolution": {
-            "startDateTime": "2017-11-10 12:00:00",
+            "startDateTime": "2017-11-10 13:00:00",
             "endDateTime": "2017-11-10 16:00:00"
           }
         },
-        "Start": 92,
-        "Length": 23
+        "Start": 63,
+        "Length": 29
       }
-    ],
-    "NotSupported": "dotNet",
-    "NotSupportedByDesign": "javascript,python,java"
+    ]
   },
   {
-    "Input": "Cortane, maak een afspraak met skype for business met Wayne op vrijdagmiddag tussen 1 en 4. ",
```

Comment:
Where these couple ones just a spec order change? If so, it would be better to not change order or to highlight which cases are just a sorting change to make review easier.
I've reviewed this file from the the bottom up and it was not 100% clear.

Reply:
yes this case is still in DateTimePeriodParser, we inserted each variation immediately after its original case since they are related, but this probably makes it difficult to compare the file changes.

Reply:
Let's make sure when editing specs to only add new cases at the end, ok?
It really greatly simplify reviews.